pkg/manifests/manifests.go

```diff
@@ -2146,3 +2123,27 @@ func (f *Factory) HashTrustedCA(caBundleCM *v1.ConfigMap, prefix string) *v1.Con
 		},
 	}
 }
+
+func trustedCABundleVolumeMount(name string) v1.VolumeMount {
+	return v1.VolumeMount{
+		Name:      name,
+		ReadOnly:  true,
+		MountPath: "/etc/pki/ca-trust/extracted/ca-bundle/",
```

Comment:
looks good, but we should verify in `#warroom-proxy` if this is a correct place.